pkg/profiler/cpu/cpu.go

```diff
@@ -244,7 +244,7 @@ func loadBpfProgram(logger log.Logger, reg prometheus.Registerer, debugEnabled,
 			break
```

Comment:
Actually, will change the log to here so we see the error every time in case it changes